MapboxCoreNavigation/RouteController.swift

```diff
@@ -118,8 +115,45 @@ open class RouteController: NSObject {
         }
     }
     
+    /**
+     IOS 8601 timestamp of when the `RouteController` was initialized.
+     */
+    public let sessionStartTimestamp: String = Date().ISO8601
```

Comment:
Same question as in https://github.com/mapbox/mapbox-navigation-ios/pull/304/files#r123724999, can make this sharable with MapboxNavigation but not public?